examples/source/interactivity-dynamic-content/Paged_List/api.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+const MAX_PAGE_COUNT = 5;
+const ITEMS_PER_PAGE = 4;
+
+class ProductListing {
+  constructor(title, image, copy) {
+    this.title = title;
+    this.image = image;
+    this.copy = copy;
+  }
+}
+
+class PagedResponse {
+  constructor(currentPage, pageCount, products) {
+    this.currentPage = currentPage;
+    this.pageCount = pageCount;
+    this.products = products;
+  }
+}
+
+class AmpListResponse {
+  constructor(items) {
+    this.items = items;
+  }
+}
+
+examples.get('/search', (request, response) => {
+  const {query} = request;
+  const page = Number(query.page) > 0 ? Number(query.page) : 1;
```

Comment:
You can simplify this to: 

```
let page = Number(request.query.page);
if (page < 0) {
  page = 1
}
```

Reply:
Alrigiht, but I also added a check if page is NaN, because by default it should return page 1, shouldn't it?

Reply:
👍

Reply:
PTAL